source/operations/install-deploy-manage/deploy-operator-helm.rst

```diff
@@ -0,0 +1,316 @@
+.. _minio-k8s-deploy-operator-helm:
+
+=========================
+Deploy Operator With Helm
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+Helm is a tool for automating the deployment of applications to Kubernetes clusters.
+A `Helm chart <https://helm.sh/docs/topics/charts/>`__ is a set of YAML files, templates, and other files that define the deployment details.
+The following procedure uses a Helm Chart to install the :ref:`MinIO Kubernetes Operator <minio-operator-installation>` to a Kubernetes cluster.
+
+
+Prerequesites
+-------------
+
+To install the Operator with Helm you will need the following:
+
+* An existing cluster using Kubernetes 1.19.0 or later (1.21.0 or later recommended).
+* The ``kubectl`` CLI tool on your local host, the same version as the cluster.
+* `Helm <https://helm.sh/docs/intro/install/>`__ version xx or greater.
+* `yq <https://github.com/mikefarah/yq/#install>`__ version xx or greater.
+* Access to run ``kubectl`` commands on the cluster from your local host.
+
+For more about Operator installation requirements, including TLS certificates, see the :ref:`Operator deployment prerequisites <minio-operator-prerequisites>`.
```

Comment:
Production deployments likely need this additional info/context from the main Operator deploy page. How much is worth explicitly referencing/duplicating here?

Reply:
I think you have sufficient information with appropriate linking of where to go for more info. It's fine from my view.